src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active nav__link--is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac 27"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/user-attachments/assets/37946f19-f3b8-4d7b-9beb-31eb660078b8)